pkg/api/server/v1alpha2/logs.go

```diff
@@ -65,7 +65,7 @@ func (s *Server) GetLog(req *pb.GetLogRequest, srv pb.Logs_GetLogServer) error {
 		return status.Error(codes.NotFound, "Log doesn't exist")
 	}
 
-	writer := logs.NewBufferedWriter(srv, req.GetName(), s.config.LOGS_BUFFER_SIZE)
+	writer := logs.NewBufferedHTTPWriter(srv, req.GetName(), s.config.LOGS_BUFFER_SIZE)
```

Comment:
Does this break `tkn-results` if it uses gRPC to receive data?

Reply:
I also tested with tkn-results, it doesn't break but the data received is not very good on the UX side, we may put that as another PR. We already have an issue around UX.

Reply:
@adambkaplan It doesn't break tkn-results. The only difference in the gRPC response in the `name` field is not there, which is anyways a part of the request.

Reply:
@avinal Are you referring to the data received in the gRPC method? If so, then it would be actually different from what you get from tkn-results. Because any client implementation of the gRPC would need to use the generated stubs to use the API, which has all the getters. But in case of tkn-results, it just prints the response proto message. To have the actual log printed, the `GetData` methods should be used.

Reply:
Got it